org-formation/075-security-hub/_tasks.yaml

```diff
@@ -0,0 +1,34 @@
+Parameters:
+  <<: !Include '../_parameters.yaml'
+
+  appName:
+    Type: String
+    Default: 'securityhub'
+
+  accountId:
+    Type: String
+    Description: The identifier from the account used to manage GuardDuty
+    Default: !Ref SecurityCentralAccount
+
+SecurityHub:
+  Type: update-stacks
+  Template: security-hub.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}'
+  StackDescription: SecurityHub notifications
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+
+SecurityHubNotifications:
+  Type: update-stacks
+  Template: notifications.njk
+  StackName: !Sub '${resourcePrefix}-${appName}-notifications'
+  StackDescription: SecurityHub notifications
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    Account: !Ref accountId
+  TemplatingContext:
+    Subscriptions:
+      # refer to https://docs.aws.amazon.com/sns/latest/api/API_Subscribe.html for valid list of protocol and endpoints
+      - Protocol: "email"
+        Endpoints: ["foo@sagebase.org","bar@sagebase.org"]
```

Comment:
Can you parameterize this and input as a var from the workflow?

Reply:
`Protocol` and `Endpoints` are vars.  These vars get passed into `notifications.njk` template,  you just need to decide what emails you want here.

Reply:
probably `it-alert-inbox@`